### PR TITLE
Cleanup strings.xml: removed duplicates, Google sign in button now us…

### PR DESCRIPTION
**Pull request by peterhav**
Changes as discussed in issue  #242, please verify @samtstern

### CODE REVIEW

**Comment by peterhav** [comment]
2016-08-09 21:52 GMT+02:00 googlebot <notifications@github.com>:

> I signed it!

File: auth/src/main/res/layout/idp_button_google.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Button
     xmlns:android="http://schemas.android.com/apk/res/android"
     style="@style/FirebaseUI.Button.AccountChooser.GoogleButton"
-    android:text="@string/sign_in_with_google"
+    android:text="@string/common_signin_button_text_long"
```
Comment (samtstern) [inline auth/src/main/res/layout/idp_button_google.xml]: Where is this defined, in Google Play Services?  I don't see it in `strings.xml`.  If it's defined in a dependency let's re-define it here so we don't get changing text over time, the string resources of Google Play Services are not considered a public API and may change or go away.

**Comment by samtstern** [comment]
@peterhav one comment, otherwise LGTM.

**Comment by samtstern** [comment]
LGTM, thanks!

**Pull request merged**